README.md

```
@@ -29,6 +29,37 @@ Discover organizations based on **tech stack, domains, competition level, GitHub
+open index.html   # macOS
+# OR double-click index.html on Windows
+```
+
+**3. Make your change & open a PR**
+- Edit files, save, refresh `index.html` in your browser to see changes
+- Follow the [GSSoC Contributor Guide](docs/GSSOC_CONTRIBUTOR_GUIDE.md) for PR rules
+
+> 💡 No Node.js, no npm, no build tools needed. This is a pure HTML/CSS/JS project.
+
+**Helpful links for new contributors:**
+| Guide | Link |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)